src/blocks/button/Button.utils.ts

```diff
@@ -197,6 +197,11 @@ export const getButtonSizeStyles = ({
       font-weight: 500;
       line-height: 16px;
 
+      .icon {
+        width: 16px;
+        height: 16px;
+      }
```

Comment:
```suggestion
     [role='img'] {
    width: 24px;
    height: 24px;
  }
```

Please try this approach because it will directly change the svg size and not the container size of the svg

Reply:
This is not working @rohitmalhotra1420

Reply:
It's working, as we discussed on the call.